modules/tinymce/src/core/main/ts/content/InsertContentImpl.ts

```diff
@@ -22,6 +22,16 @@ import * as SelectionUtils from '../selection/SelectionUtils';
 import { InsertContentDetails } from './ContentTypes';
 import * as InsertList from './InsertList';
 
+const isFlattenable = (fragment: AstNode) => {
```

Comment:
A better name for this might be `shouldFlatten`? As that's more what we're checking is if the fragment content should be flattened when merged into the existing content.

Reply:
done